packages/entrykit/playground/wagmiConfig.ts

```diff
@@ -1,34 +1,37 @@
 import { Chain, http } from "viem";
-import { anvil } from "viem/chains";
+import { sepolia, baseSepolia } from "viem/chains";
 import { createWagmiConfig } from "../src/createWagmiConfig";
 import { chainId } from "./common";
+import { pyrope } from "@latticexyz/common/chains";
 
 const chains = [
   {
-    ...anvil,
```

Comment:
can we keep anvil for the playground?

Reply:
yes, will undo, now just testing on Pyrope the whole flow so that it works with relay.link

Reply:
should we undo this now before merging?